src/devices/Bmx280/DeviceStatus.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Iot.Device.Bmx280
+{
+    public class DeviceStatus
```

Comment:
Do you mind adding doc comments to this class and to the FilteringMode as well?

Reply:
Will do.